dotnet/src/SemanticKernel/SkillDefinition/SKFunction.cs

```diff
@@ -766,7 +766,7 @@ private static void TraceFunctionTypeCall(DelegateTypes type, ILogger log)
     }
 
     [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-    private string DebuggerDisplay => $"{Name} ({Description})";
+    private string DebuggerDisplay => $"{this.Name} ({this.Description})";
```

Comment:
@shawncal looks like this type of error is not automatically detected

Reply:
Interesting... this normally is flagged. Is it not showing warnings at all, or just not auto-fixing?
We can follow up.